frontend/app/_types/inputs.ts

```diff
@@ -1,7 +1,7 @@
 export type TextInputType = {
   id: string
   placeholder: string
-  defaultValue?: string
+  value: string | number
```

Comment:
Just realized this InputType is Text specifically (hence the type name), and I won't make it a general Input type that accepts text & numbers & date etc... so I need to remove the "number" here and some of the type's types (date, number...)